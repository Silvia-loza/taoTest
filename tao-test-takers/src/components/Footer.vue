<template>
    <footer class="footer">
    <ul>
    <li>Products</li>
    <li>Solutions</li>
    <li>Services</li>
    <li>Support</li>
    <li>Company</li>
    </ul>
    </footer>
</template>

<script>
export default {
  name: 'Footer',
  props: {}
}
</script>

<style scoped>

.footer{
display:flex;
align-items: center;
justify-content: space-between;
background-color:#263e4e ;
color: white;
height: 30vh;
font-size: 3em
}

</style>
