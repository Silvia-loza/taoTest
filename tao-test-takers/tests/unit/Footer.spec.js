import { shallowMount } from '@vue/test-utils'
import Footer from '@/components/Footer.vue'

describe("Header works", () => {
    it("renders the component", () => {

        //arrange item to be mounted
        const wrapper = shallowMount(Footer)

        //assert that the component is mounted and that it matches the snapshot creted, will inform us about changes.
        expect(wrapper.html()).toMatchSnapshot()
    })
})