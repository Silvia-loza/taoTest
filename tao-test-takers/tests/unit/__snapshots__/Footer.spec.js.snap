// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header works renders the component 1`] = `
<footer class="footer">
  <ul>
    <li>Products</li>
    <li>Solutions</li>
    <li>Services</li>
    <li>Support</li>
    <li>Company</li>
  </ul>
</footer>
`;
